src/mahoji/commands/smelt.ts

```diff
@@ -120,8 +117,25 @@ export const smeltingCommand: OSBMahojiCommand = {
 				timeToSmithSingleBar *= 1.075;
 				boosts.push('-7.5% penalty for not having graceful equipped.');
 			}
+			if (user.hasItemEquippedAnywhere('Smithing master cape')) {
+				timeToSmithSingleBar /= 2;
+				boosts.push('2x boost for Smithing master cape');
+			}
+			if (user.hasItemEquippedAnywhere(itemID('Dwarven gauntlets')) && bar.id !== itemID('Gold bar')) {
+				boosts.push('2x boost for having a Dwarven gauntlets equipped.');
+				timeToSmithSingleBar /= 2;
+			}
 		}
 
+		if (!blast_furnace) {
```

Comment:
This is incorrect. 
In revision b7c0e151 (blastfurnace.ts) you can see that blast furnace DOES get 2x boost for Dwarven gauntlets EXCEPT for when smithing Gold.

Reply:
This implies everywhere thats not blast_furnance gets the boost, so this with the previous one should cover everything getting the boost except blast furnancing gold